package.json

```diff
@@ -184,6 +184,7 @@
     "nock": "8.0.0",
     "nodemon": "1.4.1",
     "react-addons-test-utils": "15.4.0",
+    "react-codemod": "github:reactjs/react-codemod#d6563d9",
```

Comment:
Is there not a published version of this? Otherwise, github links can be shorter: 
```reactjs/react-codemod#d6563d9```

https://docs.npmjs.com/files/package.json#github-urls